packages/data-broker/src/services/dhis/DhisInputSchemeResolvingApiProxy.js

```diff
@@ -222,12 +222,19 @@ export class DhisInputSchemeResolvingApiProxy {
 
     const dhisIds = response.rows.map(row => row[orgUnitIdIndex]);
 
-    const mappings = await this.models.dataServiceEntity.find({
-      [QUERY_CONJUNCTIONS.RAW]: {
-        sql: `config->>'dhis_id' in (${dhisIds.map(() => '?')})`,
-        parameters: dhisIds,
-      },
-    });
+    let mappings = [];
+    // need to batch this query because we can get an SQL error if we do too many
```

Comment:
Happy for this not to happen now as I know it's an urgent hot fix, but please come back to it

```suggestion
    // need to batch this query because we can get an SQL error if we do too many
    // TODO convert to using `runDatabaseFunctionInBatches` or one of its wrappers
```